code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -603,6 +686,9 @@ var/global/list/robot_modules = list(
 	src.modules += new /obj/item/weapon/form_printer(src)
 	src.modules += new /obj/item/weapon/gripper/paperwork(src)
 	src.modules += new /obj/item/weapon/hand_labeler(src)
+	src.modules += new /obj/item/weapon/folder/red(src) // for any remarks

```

Comment:
Test this when using the folder on a table. It may end badly with the folder being removed?